Gravity/Gravity/Base/BaseDto.cs

```diff
@@ -77,7 +77,20 @@ public static Guid GetRelativityFieldGuidOfProperty<T>(string propertyName)
 			return fieldAttribute?.FieldGuid ?? new Guid();
 		}
 
-		public PropertyInfo GetParentArtifactIdProperty()
+	    public static RdoFieldType GetRelativityFieldTypeOfProperty<T>(string propertyName)
+	    {
+	        var fieldAttribute = typeof(T).GetPublicProperties()
+	            .FirstOrDefault(property => property.Name.Equals(propertyName, StringComparison.OrdinalIgnoreCase))?
+	            .GetCustomAttribute<RelativityObjectFieldAttribute>();
+
+            //TODO: need to update this ugliness when convert Field Type to RdoFieldType
+	        int fieldTypeAsInt = fieldAttribute?.FieldType ?? -1;
+	        RdoFieldType returnType = (RdoFieldType) fieldTypeAsInt;
+
+            return returnType;
```

Comment:
Collapse with previous line

Reply:
Yeah, I like whitespace...but your right, done.

Reply:
I mean lines 88-90 should be a single line :-)